tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java

```diff
@@ -435,6 +434,40 @@ public boolean canDeleteUsers() {
         }
     }
 
+    /** See VoteResolver getStatusForOrganization to see how this is computed. */
+    public enum VoteStatus {
+        /**
+         * The value for the path is either contributed or approved, and the user's organization
+         * didn't vote. (see class def for null user)
+         */
+        ok_novotes,
+
+        /**
+         * The value for the path is either contributed or approved, and the user's organization
+         * chose the winning value. (see class def for null user)
+         */
+        ok,
+
+        /**
+         * The user's organization chose the winning value for the path, but that value is neither
+         * contributed nor approved. (see class def for null user)
+         */
+        provisionalOrWorse,
```

Comment:
The comment is wrong -- there is no implication whatsoever about the user's organization

This will be fixed in the next PR